### PR TITLE
Footer component improvement add link and css

### LINKED ISSUE #75: Storybook: Footer Component (improvement)
1) In footer i want to click on the icon and the text written on the bottom of the icon to land on that page so
both should be clickable (hyperlinked)

2) The icon images should be transparent without any background color. It can be exported from the wireframe by just double clicking on the image and selecting the export option in the right side. Please see the attached image.
exporting images

3) We can achieve the same design with less CSS, please give it a try.

### PR DESCRIPTION
**Pull request by yukafjaponesa**
# Description

footer component improvement. make clickable link to home and profile button. 
style those button.

Fixes #75 

# How to test?

story book

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/footer/footer.js [hunk 1]
```diff
@@ -19,16 +19,22 @@ const Footer = () => {
...
-          <img src={home} alt="home icon" />
-          <h4>HOME</h4>
-        </div>
-        <div className=" box">
-          <Link to={getProfileLink()}>
-            <img src={profile} alt="profile icon" />
-            <h4>PROFILE</h4>
-          </Link>
-        </div>
+        <Router>
```
Comment (tpetrina) [inline src/client/components/footer/footer.js:22]: No need to place Router here. Remove it.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Looks good, please remove extra Router.

File: src/client/components/footer/footer.js (hunk 1 above)
Comment (yukafjaponesa) [reply src/client/components/footer/footer.js:22]: @tpetrina if i remove <Router>(for Home) from Line22, i get an error as : Invariant failed: You should not use <Link> outside a <Router>. When I use Link, i need Router.

Comment (tpetrina) [reply src/client/components/footer/footer.js:22]: Where do you get this error? Is it in storybook? In that case you must use `StaticRouter` _inside_ storybook. Router is already added in `App` and doesn't need to be added twice.

Comment (Yaser-Aldammad) [reply src/client/components/footer/footer.js:22]: Yuka check my PR footer link-router. See how I added the router to storybook file and then add you part on Home in the Footer.js

Comment (tpetrina) [reply src/client/components/footer/footer.js:22]: https://github.com/HackYourFuture-CPH/web-dev-rooms/pull/223 solves this issue correctly.

Comment (yukafjaponesa) [reply src/client/components/footer/footer.js]: @Yaser-Aldammad @tpetrina  thank you so much. Please review my updated PR.

**Review by tpetrina — DISMISSED** [review]
You didn't really need to add this package, but it's fine.

**Review by Yaser-Aldammad — COMMENTED** [review]
1. got to your branch
2. go to the left corner where you can see changed files
3. choose package.json and click the arrow to Discard Changes to this particular file.
4. you are fine now and push again

**Comment by yukafjaponesa** [comment]
@Yaser-Aldammad thank you. i have discarded the changes in package.json and package-lock.json.

**Review by Yaser-Aldammad — DISMISSED** [review]

**Review by Yaser-Aldammad — APPROVED** [review]

**Review by tpetrina — APPROVED** [review]

**Pull request merged**